packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -87,7 +87,7 @@ function toPodInfo(pod: V1Pod, contextName?: string): PodInfo {
     Name: pod.metadata?.name || '',
     Namespace: pod.metadata?.namespace || '',
     Networks: [],
-    Status: pod.status?.phase || '',
+    Status: pod.metadata?.deletionTimestamp ? 'DELETING' : pod.status?.phase || '',
```

Comment:
Haha! That's great. I encountered the same issue with the k8s POC implementation.

So odd how there's no "terminating" status, just a deletionTimestamp..

LGTM. Did a quick test and it'll actually update the deletion status now.